src/zjs_modules.c

```diff
@@ -95,16 +96,26 @@ static jerry_value_t native_require_handler(const jerry_value_t function_obj,
     for (int i = 0; i < modcount; i++) {
         module_t *mod = &zjs_modules_array[i];
         if (!strcmp(mod->name, module)) {
-            return jerry_acquire_value(mod->init());
+            // We only want one intance of each module at a time
+            if (mod->instance == 0) {
+                mod->instance = jerry_acquire_value(mod->init());
+            }
+            return mod->instance;
         }
     }
 
-    PRINT("MODULE: `%s'\n", module);
     return zjs_error("native_require_handler: module not found");
 }
 
 void zjs_modules_init()
 {
+    int modcount = sizeof(zjs_modules_array) / sizeof(module_t);
+
+    for (int i = 0; i < modcount; i++) {
+        module_t *mod = &zjs_modules_array[i];
+        mod->instance = 0;
```

Comment:
It absolutely is zero from the start, no question there.  But this is for when we launch a ZJS app multiple times in ashell, I need to reset the values to zero without nuking the entire struct. I'm open to suggestions for something more efficient :)